frontend/src/styles/_responsive.scss

```
@@ -588,6 +563,9 @@
   .date {
     grid-column: 4 / 5;
     grid-row: 1;
+    // Narrow rows put the date right beside the text with no toolbar between
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)